alerting/src/test/kotlin/org/opensearch/alerting/AccessRoles.kt

```diff
@@ -30,9 +37,13 @@ val ROLE_TO_PERMISSION_MAPPING = mapOf(
     ALERTING_SEARCH_EMAIL_GROUP_ACCESS to "cluster:admin/opendistro/alerting/destination/email_group/search",
     ALERTING_INDEX_MONITOR_ACCESS to "cluster:admin/opendistro/alerting/monitor/write",
     ALERTING_GET_MONITOR_ACCESS to "cluster:admin/opendistro/alerting/monitor/get",
+    ALERTING_GET_WORKFLOW_ACCESS to AlertingActions.GET_WORKFLOW_ACTION_NAME,
```

Comment:
As a followup, we should also update the other cluster permissions to point to the action names as well.